### ISSUE TITLE
TSCANorder parameter mismatch in infer_tree_structure function

### ISSUE THREAD

**Issue by yu-tong-wang** [Winnie09/Lamian#33]
When using the `infer_tree_structure` function, I encountered an error related to the `TSCANorder` function call. The error suggests a mismatch between the parameters being passed and the actual function signature.

## Error Message
```R
Error in TSCANorder(mcl, startcluster = origin.cluster, listbranch = TRUE) : 
  unused argument (startcluster = origin.cluster)
```

This might be related to the TSCAN package code update (https://github.com/LTLA/TSCAN/blob/26abecf4bbc2785effddbb1db27c856fd236aac3/R/TSCANorder.R), where
```R
TSCANorder <- function(mclustobj,MSTorder = NULL,orderonly=T,flip=F,listbranch=F) {...}
```

**Comment by wisdomadingo** [comment]
True, `infer_tree_structure` used to work just last year December but got the same error after updating today. Which is a bit strange because I dont see any recent commits in Lamian nor TSCAN repo.

**Comment by wisdomadingo** [comment]
I have also observed that, I get the error when I run line by line but it seems to work when I knit the Rmarkdown document
